stories/components/common/stats/Stat.js

```diff
@@ -7,6 +7,9 @@ import { select, button, boolean } from '@storybook/addon-knobs';
 import Stat from '../../../../src/components/common/stat/Stat';
 import { MGDL_UNITS, MMOLL_UNITS, MS_IN_DAY, MGDL_CLAMP_TOP, MMOLL_CLAMP_TOP, MGDL_PER_MMOLL } from '../../../../src/utils/constants';
 import { getSum, statFormats, statTypes } from '../../../../src/utils/stat';
+import i18next from 'i18next';
```

Comment:
Why translate the stories ?
We don't use them (not used by Blip), and doing just this part has no meaning.

Reply:
That's true. 
let me revert that